### ISSUE TITLE
6.0.0 npm error: node_modules/ember-shepherd: Appears to be a git repo or submodule.

### ISSUE THREAD

**Issue by st-h** [shipshapecode/ember-shepherd#319]
ember-shepherd 6.0.0 fails to install via npm with the following error:
```
npm install
npm WARN deprecated exists-sync@0.0.4: Please replace with usage of fs.existsSync
npm WARN deprecated browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated exists-sync@0.0.3: Please replace with usage of fs.existsSync
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm ERR! path /Users/steve/code/myapp/myapp/node_modules/ember-shepherd
npm ERR! code EISGIT
npm ERR! git /Users/steve/code/myapp/myapp/node_modules/ember-shepherd: Appears to be a git repo or submodule.
npm ERR! git     /Users/steve/code/myapp/myapp/node_modules/ember-shepherd
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```
If one looks at the node_modules directory, it contains a `.git` directory which likely is the cause of the error. ember-shepherd 5.0.0 does not contain that directory.
```
ls -a node_modules/ember-shepherd/
.		..		.DS_Store	.git		CHANGELOG.md	HISTORY.md	LICENSE.md	README.md	addon		app		config		fastboot	index.js	package.json
```
yarn does not seem to have an issue with this and installs fine. However, it would probably be a good idea to make sure the `.git` and `.DS_Store` folder is removed from future releases.

**Comment by rwwagner90** [comment]
@st-h we haven't changed anything regarding the npm publishing. Perhaps older npm versions ignored `.git` by default or something?

**Comment by st-h** [comment]
@rwwagner90 I actually do not know. It showed up in our CI, where the ember build was still using npm. I tried to reproduce it locally and was able to do so. Then I switched back to ember-shepherd 5.0 and had a look at what got installed in the `node_modules` folder. I noticed that only `6.0` contained the `.git` directory. `5.0` does not contain `.git` or `.DS_Store` files.

If I have a look at the `node_modules` directory of my app, ember-shepherd is the only one shipping a `.git` directory:
```
$ find node_modules -name '.git'
node_modules/ember-shepherd/.git
```

I don't have a clue what might be causing this though

**Comment by rwwagner90** [comment]
@st-h I believe it is a bug when publishing packages with newer npm versions. This seems related https://npm.community/t/npm-pack-includes-items-from-git-folder-if-there-is-a-branch-called-readme-regression-in-6-9-0/6044

According to https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package `.git` should be ignored by default.

**Comment by st-h** [comment]
@rwwagner90 that sounds nasty... You are right that npm should ignore .git files by default. I am just a little puzzled that we are not seeing this with other add ons as the post you linked already is a few months old. But people should probably be using yarn by now anyways :trollface:

**Comment by rwwagner90** [comment]
@st-h it might be some sort of edge case, and not happening all the time. I may have just updated node/npm on the particular day the bug was occurring, and things are okay now. Either way, I just published 6.0.1 and use an older npm, which hopefully should fix the problem. Can you try nombom and installing 6.0.1 and see if it fixes the issue?

**Comment by st-h** [comment]
@rwwagner90 `6.0.1` looks good to me :)
```
$ ls -a node_modules/ember-shepherd/
.		..		CHANGELOG.md	HISTORY.md	LICENSE.md	README.md	addon		app		config		fastboot	index.js	package.json
```
